Src/ILGPU.Tests/MemoryBufferOperations.tt

```diff
@@ -0,0 +1,396 @@
+﻿<#@ template debug="false" hostspecific="true" language="C#" #>
+<#@ include file="Generic/ConfigurationBase.tt" #>
+<#@ assembly name="System.Core" #>
+<#@ import namespace="System.IO" #>
+<#@ output extension=".cs" #>
+using ILGPU.Runtime;
+using System;
+using System.Diagnostics.CodeAnalysis;
+using Xunit;
+using Xunit.Abstractions;
+
+<#
+var copyOperations = new string[] { "CopyFrom", "CopyTo" };
+var arrayTypes = new string[] { "Jagged", "Multidimensional" };
+#>
+namespace ILGPU.Tests
+{
+    public abstract class MemoryBufferOperations : TestBase
+    {
+        protected MemoryBufferOperations(ITestOutputHelper output, TestContext testContext)
+            : base(output, testContext)
+        { }
+
+        private static int[] InitialiseArray1D(int x)
```

Comment:
We should change "initialise" to "initialize" because the code base is written in US style.

Reply:
Done